src/services/highcharts-factory/highcharts-factory.svc.coffee

```diff
@@ -17,6 +17,16 @@ angular
           layout: 'vertical'
           align: 'left'
           verticalAlign: 'middle'
+          useHTML: true
+          plotOptions :
+             series:
+               dataLabels:
+                 useHTML: true
+          labelFormatter: ->
+            name = this.name
+            imgSrc = ImpacAssets.get(_.camelCase(name + 'LegendIcon'))
+            img = "<img src='#{imgSrc}'><br>"
+            return img + '	' + name
```

Comment:
The idea of the templates object here is just to reduce the need to define the boilerplate for each chart type, e.g 'line', 'column' etc. 

The configuration you've done here is defining a formatter, so therefore should be apart of the `formatters` instance method of the `Chart` class. 

But, as this configuration is specific to the cash-projection widget, we can override the `formatters` instance method from the cash-projection component.

accounts-cash-projection.directive.coffee:
```coffeescript
w.format = ->
    ...
    $scope.chart ||= new HighchartsFactory($scope.chartId(), w.content.chart, options)
    # Extend default chart formatters to add custom legend img icon
    defaultFormattersConfig = $scope.chart.formatters()
    $scope.chart.formatters = ->
      angular.merge(defaultFormattersConfig, {
        legend:
          useHTML: true
          labelFormatter: ->
            name = this.name
            img = "<img src='dist/images/#{name.replace(' ', '-').toLowerCase()}.png' width = '55px' height = '25px'><br>"
            console.log 'formatter! ', options
            return img + '  ' + name
      })
```